osu.Game/Screens/Multi/Multiplayer.cs

```diff
@@ -251,14 +251,11 @@ protected override void Update()
 
                 if (track != null)
                 {
+                    track.RestartPoint = Beatmap.Value.Metadata.PreviewTime;
                     track.Looping = true;
 
                     if (!track.IsRunning)
-                    {
-                        game.Audio.AddItem(track);
-                        track.Seek(Beatmap.Value.Metadata.PreviewTime);
-                        track.Start();
-                    }
+                        track.Restart();
```

Comment:
what's the scenario where this is required? we may want to not have this if it meant the user can no longer pause audio while in a lobby.

Reply:
All of this code will have to change when that is required, because it wasn't possible before and still isn't possible now. This is required for the restart point to get seeked to.

Reply:
even with looping enabled? o_O

Reply:
Yes, a seek does not happen on a simple `.Start()`

Reply:
@smoogipoo rewrote to move this logic out of update. requesting re-review.